? gtk-config-2.0
? conftest.S
? gdk/makefile.mingw
? gdk/win32/makefile.mingw
? gdk-pixbuf/makefile.mingw
? gdk-pixbuf/pixops/makefile.mingw
? gtk/makefile.mingw
? po/gtk20.pot
Index: ChangeLog
===================================================================
RCS file: /cvs/gnome/gtk+/ChangeLog,v
retrieving revision 1.1383
diff -u -u -r1.1383 ChangeLog
--- ChangeLog	2000/08/30 00:33:34	1.1383
+++ ChangeLog	2000/08/30 19:45:23
@@ -1,3 +1,13 @@
+2000-08-02  Martin Baulig  <baulig@suse.de>
+
+	* gtk/gtkbindings.c (binding_entry_destroy): Don't call
+	the deprecated g_hash_table_freeze/thaw().
+	(binding_entry_new): Likewise.
+
+	* gtk/gtkrc.c (gtk_rc_parse_file): Use g_path_get_dirname() and
+	not the deprecated g_get_dirname().
+	(gtk_rc_parse_any): Don't call g_scanner_freeze/thaw_symbol_table().
+
 2000-08-29 Elliot Lee <sopwith@redhat.com>
 	* */*.h: Use G_GNUC_CONST.
 
Index: gdk/x11/gdkimage-x11.c
===================================================================
RCS file: /cvs/gnome/gtk+/gdk/x11/gdkimage-x11.c,v
retrieving revision 1.24
diff -u -u -r1.24 gdkimage-x11.c
--- gdk/x11/gdkimage-x11.c	2000/07/26 11:32:30	1.24
+++ gdk/x11/gdkimage-x11.c	2000/08/30 19:45:24
@@ -355,7 +355,6 @@
 	  break;
 
 	case GDK_IMAGE_FASTEST:
-	case GDK_IMAGE_SHARED_PIXMAP:
 	  g_assert_not_reached ();
 	}
 
@@ -492,7 +491,6 @@
       break;
 
     case GDK_IMAGE_FASTEST:
-    case GDK_IMAGE_SHARED_PIXMAP:
       g_assert_not_reached ();
     }
 
Index: gtk/gtkbindings.c
===================================================================
RCS file: /cvs/gnome/gtk+/gtk/gtkbindings.c,v
retrieving revision 1.20
diff -u -u -r1.20 gtkbindings.c
--- gtk/gtkbindings.c	2000/07/26 11:32:42	1.20
+++ gtk/gtkbindings.c	2000/08/30 19:45:25
@@ -122,11 +122,9 @@
   binding_set->entries = entry;
 
   entry->hash_next = g_hash_table_lookup (binding_entry_hash_table, entry);
-  g_hash_table_freeze (binding_entry_hash_table);
   if (entry->hash_next)
     g_hash_table_remove (binding_entry_hash_table, entry->hash_next);
   g_hash_table_insert (binding_entry_hash_table, entry, entry);
-  g_hash_table_thaw (binding_entry_hash_table);
   
   return entry;
 }
@@ -205,10 +203,8 @@
     g_hash_table_remove (binding_entry_hash_table, entry);
   else if (begin != o_entry)
     {
-      g_hash_table_freeze (binding_entry_hash_table);
       g_hash_table_remove (binding_entry_hash_table, entry);
       g_hash_table_insert (binding_entry_hash_table, begin, begin);
-      g_hash_table_thaw (binding_entry_hash_table);
     }
 
   entry->destroyed = TRUE;
Index: gtk/gtkrc.c
===================================================================
RCS file: /cvs/gnome/gtk+/gtk/gtkrc.c,v
retrieving revision 1.63
diff -u -u -r1.63 gtkrc.c
--- gtk/gtkrc.c	2000/08/22 03:05:13	1.63
+++ gtk/gtkrc.c	2000/08/30 19:45:27
@@ -680,7 +680,7 @@
        * a stack of directory names while parsing it
        */
       rc_dir_stack = g_slist_prepend (rc_dir_stack,
- 				      g_dirname (rc_file->canonical_name));
+ 				      g_path_get_dirname (rc_file->canonical_name));
       gtk_rc_parse_any (filename, fd, NULL);
  
       tmp_list = rc_dir_stack;
@@ -1234,10 +1234,8 @@
     }
   scanner->input_name = input_name;
 
-  g_scanner_freeze_symbol_table (scanner);
   for (i = 0; i < n_symbols; i++)
     g_scanner_add_symbol (scanner, symbols[i].name, GINT_TO_POINTER (symbols[i].token));
-  g_scanner_thaw_symbol_table (scanner);
   
   done = FALSE;
   while (!done)
